.github/workflows/internal_release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    tags-ignore:
```

Comment:
This should allow us to perform the secondary push (to reset the major tag) without triggering another pipeline build